conf/application.conf

```diff
@@ -104,6 +104,11 @@ microservice {
         port = 9583
       }
 
+      pdf_generator_service {
```

Comment:
for consistency with other services, use `-` rather than `_`

Reply:
Done